### PR TITLE
FIXUP for PR 1915 (FreeIPA) by fiyyyu

### PR DESCRIPTION
**Pull request by maximiliankolb**
This PR is based on #1915 which I have rebased to master and added an additional commit (squash before merging!) to fix the build.

@fiyyyu Please cherry-pick my commit to your PR and ask for a re-review.

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.6/Katello 4.8
* [x] Foreman 3.5/Katello 4.7 (planned Satellite 6.13)

### CODE REVIEW

**Comment by fiyyyu** [comment]
Hello @maximiliankolb 
I am trying to cherry-pick but it gives me an error:
git cherry-pick e6de27333aa6c32ccc6dfe7a8e80125d9160d1dc
fatal: bad object e6de27333aa6c32ccc6dfe7a8e80125d9160d1dc

I have followed the guidelines in https://docs.engineering.redhat.com/pages/viewpage.action?spaceKey=SAT6&title=Authoring+in+the+upstream, ie run all the following commands before trying to cherry-pick:

    git checkout master
    git pull
    git checkout [my-branch-name]
    git rebase master

What else can I do to be able to cherry-pick your commit?

**Comment by maximiliankolb** [comment]
Did you add my fork as remote to your local git repository?

**Comment by fiyyyu** [comment]
Thank you for the advice, I have now successfully cherry-picked your https://github.com/theforeman/foreman-documentation/commit/e6de27333aa6c32ccc6dfe7a8e80125d9160d1dc commit into my Satellite_IdM_auth_support_12891 branch. I hope this was the right thing to do. 
As I don't have the merge rights, I do not exactly follow how I could cherry-pick your commits into specifically:
- Foreman 3.6/Katello 4.8
- Foreman 3.5/Katello 4.7 (planned Satellite 6.13)
- AFAICS, the merge request for Satellite_IdM_auth_support_12891 is only targeted at Foreman 3.5/Katello 4.7 (planned Satellite 6.13) - https://github.com/theforeman/foreman-documentation/pull/1915/

**Comment by maximiliankolb** [comment]
I've left a comment on #1915 on how to move forward. The person that merges your PR will also do the cherry-picking to already released versions, for example Foreman 3.6/3.5.

File: guides/common/modules/proc_using-freeipa-credentials-to-log-in-to-the-project-hammer-cli.adoc [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+This section describes how to log in to your {Project} Hammer CLI with your {FreeIPA} (IdM) login and password.
+
+.Prerequisites
+* You have enrolled your {ProjectServer} into {FreeIPA} and configured it to use {FreeIPA} for authentication.
+More specifically, you have enabled access both to the {ProjectWebUI} and the {Project} API.
+ifndef::orcharhino[]
+For more information, see {InstallingServerDocURL}Using_FreeIPA_{project-context}[Using {FreeIPA}] in _{InstallingServerDocTitle}_.
+endif::[]
+* The host on which you run this procedure is configured to use {FreeIPA} credentials to log users in to your {Project} Hammer CLI.
```
Comment (mjivraja) [inline guides/common/modules/proc_using-freeipa-credentials-to-log-in-to-the-project-hammer-cli.adoc:12]:
```suggestion
* The host on which you run this procedure is configured to use {FreeIPA} credentials to log users into your {Project} Hammer CLI.
```

**Comment by maximiliankolb** [comment]
https://github.com/theforeman/foreman-documentation/pull/1915 has been merged, so this PR is no longer necessary.

**Pull request closed without merging**